README.md

```diff
@@ -12,24 +12,47 @@
         --schema-handler-package "<LINK_TO_S3>"
     ```
 
+2. View the version of the newly registered resource by running the following in your terminal:
+
+    ```shell
+    aws cloudformation list-type-versions \
+    --region "<REGION>" \
+    --type RESOURCE \
+    --type-name "<DATADOG_RESOURCE_NAME>"
+    ```
+
+3. Set this newly registered version as the `default` by running the following in your terminal:
+
+    ```shell
+    aws cloudformation set-type-default-version \
+        --region "<REGION>" \
+        --type RESOURCE \
+        --version-id <VERSION> \
+        --type-name "<DATADOG_RESOURCE_NAME>" \
+    ```
+
     With the following placeholders:
     * `<REGION>`: Your AWS region.
     * `<DATADOG_RESOURCE_NAME>`: The name of the resource to register, refer to the table below to see the supported resources.
     * `<LINK_TO_S3>`: S3 link to the resource.
-      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_FOLDER>/<RESOURCE_FOLDER>-<VERSION>.zip`
+      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_NAME>/<RESOURCE_NAME>-<VERSION>.zip`
+      * See the `Resources available` section for example links.
```

Comment:
```suggestion
      * See the [Changelog][15] for examples of the latest supported S3 links.
```

Reply:
Actually, what do we think about also linking to the table here and referencing the `Resource Link`. That way its only 1 click instead of 2 (navigating to root changelog -> individual changelog)

Reply:
updated my suggestion